scipy/sparse/tests/test_base.py

```diff
@@ -4667,18 +4667,18 @@ def test_resize_blocked(self):
                       [3, 0, 0, 0]])
         S = self.spcreator(D, blocksize=(1, 2))
         assert_(S.resize((3, 2)) is None)
-        assert_array_equal(S.A, [[1, 0],
+        assert_array_equal(S.toarray(), [[1, 0],
```

Comment:
Here and else where: the addition of `toarray()` means the array is no longer correctly aligned

Reply:
Thanks! They're realigned.